src/lib/loaders/loaders_with_authentication/causality.ts

```diff
@@ -0,0 +1,44 @@
+import urljoin from "url-join"
+import fetch from "node-fetch"
+import config from "config"
+import { causalityJwt } from "schema/v2/system/causality_jwt"
+
+const { CAUSALITY_API_BASE } = config
+const uri = urljoin(CAUSALITY_API_BASE, "graphql")
+
+interface GraphQLArgs {
+  query: string
+  variables: any
+}
+
+export default (_accessToken, userID) => {
+  const causalityLoader = ({
```

Comment:
a causality graphql data loader that just creates the causality token and sends a `fetch` request to it. This might not be necessary or good to expose to other resolvers so we could define it elsewhere if we want.